astropy/io/fits/tests/test_connect.py

```diff
@@ -92,13 +92,14 @@ def test_simple_noextension(self, tmpdir):
         t2 = Table.read(filename)
         assert equal_data(t1, t2)
 
-    def test_with_units(self, tmpdir):
+    @pytest.mark.parametrize('table_type', (Table, QTable))
```

Comment:
Tests are failing as you also have to import `QTable` at the top (so I cancelled all the other builds).

(Aside: it is very helpful to set up your own installation, so you can run tests locally; you can then also run specific tests with
```
python3 setup.py test -t astropy/io/fits/tests/test_connect.py
```